infra/spot_lambdas/start.py

```diff
@@ -22,13 +22,6 @@ def lambda_handler(event, context):
             "LaunchTemplateId": launch_template_id,
             "Version": "$Latest"
         },
-        InstanceMarketOptions={
-            "MarketType": "spot",
-            "SpotOptions": {
-                "SpotInstanceType": "one-time",
-                "InstanceInterruptionBehavior": "terminate",
```

Comment:
Out of curiosity, would there any other behavior that make sense here, and would avoid restarting from zero?

Reply:
The only other option is to stop hibernate the instance, I think. But this will still cause the GitHub job to be canceled and we'd still need some way to claen up stale instances and retry automatically. Also, with the GitHub job canceled, we'd restart from scratch anyway.